ldap/servers/slapd/csngen.c

```diff
@@ -121,6 +123,7 @@ csngen_new(ReplicaId rid, Slapi_Attr *state)
     _csngen_init_callbacks(gen);
 
     gen->state.rid = rid;
+    gen->gettime = slapi_clock_gettime;
```

Comment:
Few lines below, 'sampled_time' is set with slapi_current_utc_time.
slapi_clock_gettime and slapi_current_utc_time are both calling CLOCK_REALTIME.
It would be good if the name 'slapi_clock_gettime' would be something like 'slapi_clock_utc_gettime' so that we know call the same underlying function.